src/Spryker/Client/OauthPermission/OauthPermission/OauthPermissionReader.php

```diff
@@ -85,10 +85,16 @@ protected function extractPermissionsFromOauthToken(OauthAccessTokenDataTransfer
     /**
      * @param string $authorizationToken
      *
-     * @return array
+     * @return string
```

Comment:
```suggestion
     * @return string|null
```
based on your changes.